src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,43 @@
+import cn from 'classnames';
+import { FilterType } from '../types/FilterType';
+
+type Props = {
+  status: FilterType,
+  onStatusChange: (filter: FilterType) => void,
+};
+
+export const TodoFilter: React.FC<Props> = ({ status, onStatusChange }) => {
+  return (
+    <nav className="filter">
+      <a
+        href="#/"
+        className={cn('filter__link', {
+          selected: status === FilterType.All,
+        })}
+        onClick={() => onStatusChange(FilterType.All)}
```

Comment:
consider to make one handler factory for all filter buttons and just call on each button with proper value
```
const handleStatusChange = (filterType: FilterType) => () => onStatusChange(filterType);
```
```suggestion
        onClick={handleStatusChange(FilterType.All)}
```